Chapter12/recipe-02/cxx-example/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# set minimum cmake version
+cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
+
+# project name and language
+project(recipe-02 CXX)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_EXTENSIONS OFF)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+add_library(sum_integers sum_integers.cpp)
+
```

Comment:
Just wondering if Catch can be installed as an external project. I would keep the recipe as-is anyway, because it shows one of the ways to include external source code in a project, which is something we want to discuss later.

Reply:
It definitely can but also my reasoning was to show a way where the external sources are part of the project which is easy for Catch since it is only one file but becomes a bit hard for GTest since there is a lot so I wanted to show the two approaches. Of course I still need to discuss this in the text.